src/components/twilio-voice-basic-call-control/public/twilio-voice-basic-call-control.js

```diff
@@ -1 +1,110 @@
-// TODO: Implement Web Component
+class TwilioVoiceBasicCallControl extends HTMLElement {
+  #call;
+  #callSid;
+  #conferenceSid;
+
+  constructor() {
+    super();
+    this.attachShadow({ mode: 'open' });
+
+    this.#render();
+    this.#showButtons();
+
+    const twilioVoiceDialer = document.querySelector('twilio-voice-dialer');
```

Comment:
This will query the whole document. We should only query the child nodes of this component. Can you try `this.shadowRoot.querySelector`?

Reply:
The issue here is that `this.shadowRoot` is scoped to this component. So it has no reference of parent element, where the event is dispatched. Hence why we need to query the whole document for the parent element.

Reply:
I was confused for a bit. Yeah this is the child element, and the dialer is the parent. In this case, this is still a problem because what if you have multiple dialers in the document?

Reply:
Valid point. Ok, created this follow up ticket to "pass down UUID from Dialer to child components"

https://twilio-engineering.atlassian.net/browse/VBLOCKS-3743

Reply:
How about use https://developer.mozilla.org/en-US/docs/Web/API/Node/parentElement?

Reply:
@charliesantos 1 line fix: https://github.com/twilio/twilio-voice-js-reference-components/pull/20

Reply:
Great! Approved. I remember you created a jira ticket to pass the id. Can we close that now since it's not applicable anymore?

Reply:
yup closed it